common/aepEdgeClient.js

```diff
@@ -191,6 +191,53 @@ function createAepEdgeClient(
       });
   }
 
+  function collect(requestBody, requestHeaders = {}) {
+    const requestId = uuidv4();
+
+    let domain = edgeDomain;
+    let region = aepEdgeCluster;
+
+    if (edgeDomain === "server.adobedc.net") {
+      if (aepEdgeCluster.length > 0) {
+        domain = `${aepEdgeCluster}.${edgeDomain}`
+      }
+      region = ""
+    }
+
+    const requestUrl = [
+      `https://${domain}`,
+      edgeBasePath,
+      region,
+      "v2",
+      `collect?dataStreamId=${dataStreamId}&requestId=${requestId}`,
```

Comment:
I was surprised to see that the `collect` method has more to it besides simply using `collect?dataStreamId...` instead of `interact?dataStreamId...`.  Collect does not support specifying the cluster the way `interact` does?

The way you have done it works great, but ideally the `interact` and `collect` methods could each defer to a common method (perhaps named `edgeRequest`) specifying the subtle difference in the request.  this would maximize code reuse.

Reply:
> Collect does not support specifying the cluster the way interact does?

It does, the difference comes from the fact that another endpoint is used, `server.adobedc.net` which differs in specifying the cluster.

> The way you have done it works great, but ideally the interact and collect methods could each defer to a common method (perhaps named edgeRequest)

Totally agree, will do the change 👍